lambdas/backend/routes/feedback.js

```diff
@@ -5,32 +5,25 @@ const util = require('../util')
 
 const feedbackEnabled = !!process.env.FeedbackSnsTopicArn
 
-exports.get = (req, res) => {
+exports.get = async (req, res) => {
   console.log(`GET /feedback for Cognito ID: ${util.getCognitoIdentityId(req)}`)
 
   if (!feedbackEnabled) {
     res.status(401).json('Customer feedback not enabled')
   } else {
-    feedbackController.fetchFeedback()
-      .then(feedback => {
-        res.status(200).json(feedback)
-      })
-      .catch(err => {
-        console.log(`error: ${err}`)
-        res.status(500).json(err)
-      })
+    const feedback = await feedbackController.fetchFeedback()
+    res.status(200).json(feedback)
   }
```

Comment:
No 5xx needed or is this caught elsewhere?

Reply:
It's caught [here](https://github.com/awslabs/aws-api-gateway-developer-portal/blob/master/lambdas/backend/index.js#L24), and uncaught errors and rejections are forwarded as 500 errors by AWS Lambda itself (and logged appropriately).

Reply:
Perfect.